### ISSUE TITLE
Select does not work with VoiceOver

### ISSUE THREAD

**Issue by Basti237** [jshjohnson/Choices#1416]
**Describe the bug**
The Singe Select Input (e.g., at https://choices-js.github.io/Choices) cannot be operated using iOS VoiceOver. Although you can navigate to the Select menu—and it is read aloud and opens—you cannot then use right-swipe gestures to navigate to the individual options. This makes it difficult for people with disabilities to have the options read aloud and ultimately make a selection.
Instead, it jumps to the next element and reads that one aloud.

**To Reproduce**

1. Go to 'https://choices-js.github.io/Choices'
2. Scroll down to 'Single select input'
3. Activate VoiceOver
4. Trying to have the individual options read aloud using swipe gestures

**Expected behavior**
Each option could be visited via gestures vie VoiceOver.

**Desktop (please complete the following information):**
 - iOS

**Smartphone (please complete the following information):**
 - Device: iPhone 15
 - OS: iOS 26.5.2
 - Browser Safari
